pkg/ui/src/protobufInit.ts

```diff
@@ -1,5 +1,3 @@
-/// <reference path="../node_modules/protobufjs/stub-node.d.ts" />
```

Comment:
Not entirely sure why this needed to go, but `make lint` was complaining about it.

Reply:
Looks like they cleaned some stuff up: https://github.com/dcodeIO/protobuf.js/commit/812b38ddabb35e154f9ff94f32ad8ce2a70310f1